### ISSUE TITLE
Accessibility: Scan mode not functioning as expected on NVDA or Narrator

### ISSUE THREAD

**Issue by corinagum** [microsoft/BotFramework-WebChat#2273]
## Version

4.5.1

## Describe the bug

In Narrator and NVDA, users are unable to navigate through the transcript using AT navigation in scan mode. This makes it impossible to go through the transcript after the screen reader has read out the original received activities. 

## Expected behavior
Scan mode working as expected is minimal functionality necessary to be able to review the transcript.

## Additional context
Related to Ninja Camp :)

[Bug]
